python/ray/_raylet.pyx

```diff
@@ -920,8 +920,7 @@ cdef execute_task(
                 # log prefix with the full repr of the actor. The log monitor
                 # will pick up the updated token.
                 if (hasattr(actor_class, "__ray_actor_class__") and
-                        "__repr__" in
-                        actor_class.__ray_actor_class__.__dict__):
+                        actor_class.__ray_actor_class__.__repr__ != object.__repr__):
```

Comment:
Dies __repr__ always guarantee to exist in all classes? Should we check the hasattr?

Reply:
Since all classes inherit from `object`, `__repr__` is always there.